repos/system_upgrade/el7toel8/actors/vsftpdconfigcheck/libraries/library.py

```diff
@@ -0,0 +1,23 @@
+from leapp.libraries.common.reporting import report_with_links
+from leapp.libraries.common.tcpwrappersutils import config_applies_to_daemon
+
+
+def check_config_supported(tcpwrap_facts, vsftpd_facts):
+    bad_configs = []
```

Comment:
nit: I'd rather use list comprehension here as well instead of L6-L9
```bad_configs = [config.path for config in vsftpd_facts.configs if config.tcp_wrappers]```

Reply:
Absolutely, thanks. Done. I've also extended the test coverage a bit.